### ISSUE TITLE
Rotating proxies

### ISSUE THREAD

**Issue by nikushakhabu** [V4NSH4J/discord-mass-DM-GO#235]
I have one IP and password for my rotating proxy which sets new IP on every new request, should I only put 1 proxy in proxies.txt or should  I copy and paste it many times?

Also, with rotating proxy should I have disable_keep_alievs as true or false?

**Comment by V4NSH4J** [comment]
Yeah! Shouldn't make a difference, you can paste it once. Keep alives true better for rotating proxy... Should allow for better IP rotation

**Comment by nikushakhabu** [comment]
@V4NSH4J so disable_keep_alives right? or enable I am a bt confused on wording sorry.

**Comment by V4NSH4J** [comment]
> @V4NSH4J so disable_keep_alives right? or enable I am a bt confused on wording sorry.

Yes "disable_keep_alives" : false in config

**Comment by dtsrgb** [comment]
> disable_keep_alives

Don't have this field in the latest version? Name changed or whole field deleted?
